release-schema.json

```diff
@@ -86,6 +86,56 @@
             "string",
             "null"
           ]
+        },
+        "hasExclusiveRights": {
+          "title": "Has exclusive rights",
+          "description": "The procuring entity grants the contractor the right to perform the public service described in the contract free of competition.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "exclusiveRights": {
+          "title": "Exclusive rights",
+          "description": "Specification of the exclusive rights granted by the procuring entity to the contractor.",
```

Comment:
We can reuse language from the regulation: "The nature and extent of the exclusive rights granted …"

Reply:
Done